drivers/serial/Kconfig.nx

```diff
@@ -0,0 +1,10 @@
+# NanoXplore UART
+
+# Copyright (c) 2022, NanoXplore
+# SPDX-License-Identifier: Apache-2.0
+
+config UART_NX_NGULTRA
+	bool "NanoXplore UART driver"
+	select SERIAL_HAS_DRIVER
```

Comment:
We are moving to enable drivers based on devicetree, so please add something like this:
```
        default y
        depends on DT_HAS_NANOXPLORE_NGULTRA_UART_ENABLED
```
(See other Kconfig.* for examples)

Reply:
Done